sdk/storage/storage-blob/review/generated/src/models/index.d.ts

```diff
@@ -0,0 +1,6713 @@
+/// <reference types="node" />
+import { ServiceClientOptions } from "@azure/core-http";
+import * as coreHttp from "@azure/core-http";
+/**
+ * Key information
+ */
+export interface KeyInfo {
+    /**
+     * The date-time the key is active in ISO 8601 UTC time
+     */
+    start: string;
+    /**
+     * The date-time the key expires in ISO 8601 UTC time
+     */
+    expiry: string;
+}
+/**
+ * A user delegation key
+ */
+export interface UserDelegationKey {
+    /**
+     * The Azure Active Directory object ID in GUID format.
+     */
+    signedOid: string;
+    /**
+     * The Azure Active Directory tenant ID in GUID format
+     */
+    signedTid: string;
+    /**
+     * The date-time the key is active
+     * **NOTE: This entity will be treated as a string instead of a Date because the API can
```

Comment:
I think it's important to mention the format here. Is this millisecond ticks since the epoch? microsecond? nanosecond?

Reply:
signedTenantId instead of signedTid.
objectId instead of Oid.

Reply:
signedTid/signedOid renaming done in https://github.com/Azure/azure-sdk-for-js/pull/5450